notifications_utils/recipients.py

```diff
@@ -435,6 +435,11 @@ def get_billable_units_for_prefix(prefix):
     return INTERNATIONAL_BILLING_RATES[prefix]['billable_units']
 
 
+def use_numeric_sender(number):
+    prefix = get_international_prefix(normalise_phone_number(number))
+    return INTERNATIONAL_BILLING_RATES[prefix]['attributes']['alpha'] == 'NO'
```

Comment:
Optional: We may want to consider returning True for countries that have `REG` support as I don't know if they will get through or not on an alpha sender.

@yahoopete do you know please?

Reply:
I also wonder if we can get some more up to date data from our SMS providers about what is supported for each country as that file was last updated 4 years ago